regression/cbmc/pragma_cprover3/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--pointer-primitive-check
+^\*\* 0 of 2 failed
```

Comment:
This pattern seems a bit fragile -- the `2` checks come from `malloc` but if the library implementation changes, that number could be different in the future.

I think just checking `VERIFICATION SUCCESSFUL`, as you are already doing below, should suffice?